client/helpers/get-workflow-input-display.js

```diff
@@ -0,0 +1,19 @@
+export default input => {
+  if (!input) {
+    return input;
+  }
+
+  let parsed = {};
+
+  if (input.jsonStringFull) {
+    parsed = JSON.tryParse(input.jsonStringFull);
+  } else if (input.payloads !== undefined) {
+    parsed = input;
+  } else {
+    return input;
```

Comment:
Do you want to check `encoding` in `metadata`?

Reply:
that was actually an older code from initial implementation that i didn't get rid off. Removed `jsonStringFull` lines. There is no currently special handling of `payload.data` depending on type, just taking `data` as is and showing it